src/components/v5/shared/BurgerMenu/BurgerMenu.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { FC } from 'react';
 
-import Icon from '~shared/Icon/Icon';
+import Icon from '~shared/Icon';
 import { BurgerMenuProps } from './types';
 
-const displayName = 'v5.shared.BurgerMenu.BurgerMenu';
+const displayName = 'v5.shared.BurgerMenu';
```

Comment:
const displayName = 'v5.BurgerMenu'

All components in shared directory don't have a 'shared' prefix in display name

Reply:
Fixed